include/OsmAndCore/Map/AtlasMapRenderer_Metrics.h

```diff
@@ -78,7 +78,7 @@ namespace OsmAnd
 
             OsmAnd__AtlasMapRenderer_Metrics__Metric_renderFrame__FIELDS(EMIT_METRIC_FIELD);
 
-            virtual QString toString(const bool shortFormat = false, const QString& prefix = QString::null) const;
+            virtual QString toString(const bool shortFormat = false, const QString& prefix = QString()) const;
```

Comment:
Why do you use `{}` in one place and `QString()` in another?

Reply:
In most cases, these are equivalent entries.
Generally I use `{}`, `QString()` - was used accidentally.
Let;s negotiate which one will use by default in the scope of project to keep code in the consistent state.